src/org/zaproxy/zap/extension/saml/resources/Messages.properties

```diff
@@ -56,4 +56,9 @@ saml.editor.relaystate=Relay State
 saml.editor.tab.request=Request
 saml.editor.msg.resetfailed=Resetting failed
 saml.editor.msg.confirmreset=Are you sure to reset settings?
-saml.editor.title=SAML Request Viewer
\ No newline at end of file
+saml.editor.title=SAML Request Viewer
+
+saml.passivescanner.name=Use of SAML
+saml.passivescanner.desc=The application use SAML
+saml.passivescanner.soln=Manually confirm that SAML is used, and then configure attacks in the SAML Extension (Tools > SAML Settings) to check if its correctly implemented
```

Comment:
- `it's`
 - This seems more appropriate for the description or other info field.